client/dist/style.css

```diff
@@ -646,6 +646,131 @@
   border: 1px solid #17E9E0;
 }
 
+/* Questions and Answers components ================================ */
+
+/* QA Container styles */
+.questions-answers-container, .search-questions-container, .questions-list-container, .question-container, .question-form-container, .answer-form-container, .answers-list-container, .answer-container {
```

Comment:
When editing so many different HTML elements, I think this reads a little easier when they each get their own line, instead of all being in one line.

Reply:
Went ahead and broke all of these CSS selectors onto separate lines.